src/main/groovy/com/athaydes/spockframework/report/internal/ProblemBlockWriter.groovy

```diff
@@ -45,6 +54,19 @@ class ProblemBlockWriter {
         }
     }
 
+    void skipsContainer( MarkupBuilder builder, Runnable createProblemList ) {
```

Comment:
I think it would be much better to just treated skipped tests as we treat ignored tests. I don't see why we would want to show skipped tests differently.

Reply:
Can you please point me to the right direction please? Do you expect this to be part of `writeFeatureDescription`?

Reply:
I mean that you don't need to change anything other than add a new method, `iterationSkipped`, to the spock-reports extension. Also, just consider any test that throws `AssumptionViolationException` as "skipped". No changes are required to the methods that build the xml.